### ISSUE TITLE
Saving a selection of an unsaved search to a list is broken

### ISSUE THREAD

**Issue by camerojo** [talentbeyondboundaries/tbbtalentv2#731]
Saving a selection on an unsaved search not only saves the selection but also saves the search with the same name as the list.

**Comment by camerojo** [comment]
Change the status of this issue from being labelled as a bug. And moving back to Product backlog.


Actually this is working as designed. Saving a selection from an unsaved search needs to save the search - which ii does using the name of the list.

It might be better for to do display a pop up explaining that the search has been autosaved.

It would also be good to make the Save Again/Replace Again functionality work in this case. Currently it doesn't.

In order to do that, it might be worth considering using the currently unused Saved Search database field default_save_selection_list_id to record the current default list. Currently that is is just stored in browser memory and gets lost on a refresh.

**Comment by sadatmalik** [comment]
@camerojo @cazcam34 I think this issue can be closed now as it has been resolved in release v2.2.3?

**Comment by cazcam34** [comment]
@sadatmalik yes this was addressed in the prior release, along with other saved search improvements see here (https://talent-catalog.github.io/talentcatalog/v223/search_functionality) Will close this issue.
